### DISCUSSION TITLE
osu!lazer fullscreen mode minimizes the game

### DISCUSSION THREAD

**Discussion by Rekiyaa** [ppy/osu#15348] [Q&A]
When i alt tab in fullscreen or change from windowed/borderless to fullscreen the game tabs out and i can't alt tab back in properly. Need to spam f11 while holding alt tab.
Restarting the game while in fullscreen mode safes it - opens in fullscreen again, just can't tab out otherwise it repeats.
Pretty sure (not 100%) that this happened after the last 1-3 updates.

Video for review: https://www.youtube.com/watch?v=jw_B3ZiZYfQ

[database.log](https://github.com/ppy/osu/files/7436027/database.log)
[network.log](https://github.com/ppy/osu/files/7436029/network.log)
[performance.log](https://github.com/ppy/osu/files/7436030/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7436031/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7436032/updater.log)

**Comment by bdach** [comment]
we've bumped SDL libraries last release, so if anything was to cause this, it's probably that. does this happen on 2021.1016.0?

other than that, do you have multiple screens/displays or just the one?

**Reply by Rekiyaa** [reply]
I have one screen.
You are right, it seems to be last release, 2021.1016.0 works perfectly.

**Comment by peppy** [comment]
Would be appreciated if anyone else can try to reproduce this one. Need to firstly figure if this is affecting all windows users or if there are more conditions required to cause the issue to arise.

**Reply by Rekiyaa** [reply]
Seems like restarting my pc fixed it for some reason, because today it works as usual. 
Didn't update anything, i guess it was a one time thing.

Sorry for making a post, without even trying to fix it with a simple pc restart.

**Reply by peppy** [reply]
That's good to hear. Please let us know if it happens again.

**Reply by Rekiyaa** [reply]
It actually happens again. I really don't know why, i didn't change/open anything since i said it works. PC restart didn't work this time.
If i can provide any information that may help, just tell me. 

if needed:
[database.log](https://github.com/ppy/osu/files/7442228/database.log)
[network.log](https://github.com/ppy/osu/files/7442230/network.log)
[performance.log](https://github.com/ppy/osu/files/7442231/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7442232/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7442233/updater.log)
